freqtrade/main.py

```diff
@@ -42,7 +40,7 @@ def _process() -> bool:
                     Trade.session.add(trade)
                     state_changed = True
                 else:
-                    logging.info('Got no buy signal...')
+                    logger.info('Got no buy signal...')
```

Comment:
I think now would be a good chance to change this to something for informative. Like, "Checked all whitelisted currencies. Found no suitable entry positions for buying. Will keep looking."

Reply:
Also the line 

```
         logger.info('No open orders found and trade is fulfilled. Marking %s as closed ...', trade)
```

is something that I feel could be explained better. At least that like: "Marking %s as closed as the trade is fulfilled and found no open orders for it."